### PR TITLE
[Microsoft.ResourceHealth] Add applicableScenarios property to metadata entity properties (API is not live yet)

### PR DESCRIPTION
**Pull request by mudit794**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by mudit794** [comment]
Added alertable in other PR - https://github.com/Azure/azure-rest-api-specs/pull/6067

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json [hunk 1]
```diff
@@ -1210,6 +1210,10 @@
             "type": "string"
           }
         },
+        "isAlertable": {
```
Comment (RyanBensonMSFT) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json]: same comment as in https://github.com/Azure/azure-rest-api-specs/pull/6067

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json [hunk 2]
```diff
@@ -1210,6 +1210,13 @@
             "type": "string"
           }
         },
+        "applicableScenarios": {
+          "description": "The list of scenarios applicable to this metadata entity.",
+          "type": "array",
+          "items": {
+            "type": "string"
```
Comment (RyanBensonMSFT) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json]: This should be an enum so the SDK can generate strongly typed values for it.

Comment (mudit794) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json]: Made changes. I will make changes to other PR once this looks good to you.

**Review by ryansbenson — COMMENTED** [review]
Signed off from ARMs side

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**